.gitignore

```diff
@@ -38,6 +38,7 @@ build/
 
 # generated source files
 src/generated/
+src/main/resources/swagger-ui.properties
 src/main/resources/version.properties
```

Comment:
Maybe not for this PR, but these lines could be removed if we instead generated these files in the build output directory instead of in the source tree: `build/resources/main/swagger-ui.properties`

Reply:
I like your approach better - I was following the current `version.properties` pattern.

Reply:
I'm struggling with this suggestion. The `compileJava` task clears out this directory and I am not sure of a way to hook into the gradle build process to make this suggestion happen. Since we're already doing this for the version file, I suggest we target this approach in a separate task.